lib/sqlalchemy/ext/asyncio/session.py

```diff
@@ -287,6 +314,33 @@ async def stream(
         )
         return _result.AsyncResult(result)
 
+    async def stream_scalars(
+        self,
+        statement,
+        params=None,
+        execution_options=util.EMPTY_DICT,
+        bind_arguments=None,
+        **kw
+    ):
+        """Execute a statement and return a stream of scalar results.
+
```

Comment:
**mike bayer** (zzzeek) wrote:

add

    :return: an :class:`_asyncio.AsyncScalarResult` object

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3065